ArduCopter/GCS_Copter.cpp

```diff
@@ -103,6 +103,11 @@ void GCS_Copter::update_vehicle_sensor_status_flags(void)
         break;
     }
 
+    // Mark motors unhealty if one of the motors is not producing thrust
+    if (copter.motors->get_lost_motor()) {
+        control_sensors_health &= ~MAV_SYS_STATUS_SENSOR_MOTOR_OUTPUTS;
```

Comment:
we need a new bit, SENSOR_MOTOR_OUTPUTS is used for safety switch

Reply:
done